Venter/models.py

```diff
@@ -24,6 +32,14 @@ def __str__(self):
 
 
 class Profile(models.Model):
+    """
+    A Profile associated with an existing user.
+    Eg: organisation name and phone number are some profile details associated with user_1
+
+    # Create a user profile
+    >>> prof_1 = Profile.objects.create(user=user_1, organisation_name="abc", profile_picture="image2.png")
+    >>> prof_2 = Profile.objects.create(user=user_2, organisation_name="abc", phone_number="9999999999")
```

Comment:
Same as the organisation model, you don't need to write sample code for this. In fact, you don't need elaborate docstrings for models. Just a one line explanation of what they are and what they're accomplishing is fine.

Reply:
Same as above